distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -116,6 +116,10 @@ Data can be ingested via ${catalog-ui}.
 . Drag and drop file(s) or click to open a navigation window.
 . After selecting the file(s) to be uploaded, select *Start* to begin uploading.
 
+Files will be processed individually with a visual cues and messaging indicating the successful resolution of each ingest operation.
```

Comment:
Use active voice, present tense. 
Ingest operation is ambiguous. Could apply to one file or all the files.
Parallelism with line 117 "file(s)"

"${branding} uploads the file(s) and indicates if the file(s) is successfully uploaded"